release-notes/7.0/preview/api-diff/rc1/Microsoft.NETCore.App/7.0-rc1_System.Security.Cryptography.md

```diff
@@ -0,0 +1,125 @@
+# System.Security.Cryptography
+
+``` diff
+ namespace System.Security.Cryptography {
+     public abstract class Aes : SymmetricAlgorithm {
+-        public static new Aes Create();
++        [UnsupportedOSPlatformAttribute("browser")]
++        public static new Aes Create();
+     }
+     [ObsoleteAttribute("Derived cryptographic types are obsolete. Use the Create method on the base type instead.", DiagnosticId="SYSLIB0021", UrlFormat="https://aka.ms/dotnet-warnings/{0}")]
+     public sealed class AesCryptoServiceProvider : Aes {
+-        public AesCryptoServiceProvider();
++        [UnsupportedOSPlatformAttribute("browser")]
++        public AesCryptoServiceProvider();
+     }
+-    [ObsoleteAttribute("Derived cryptographic types are obsolete. Use the Create method on the base type instead.", DiagnosticId="SYSLIB0021", UrlFormat="https://aka.ms/dotnet-warnings/{0}")]
+-    public sealed class AesManaged : Aes
++    [ObsoleteAttribute("Derived cryptographic types are obsolete. Use the Create method on the base type instead.", DiagnosticId="SYSLIB0021", UrlFormat="https://aka.ms/dotnet-warnings/{0}")]
```

Comment:
Why do these `ObsoleteAttribute` show up in the diff? They are identical. Only the `UnsupportedOSPlatformAttribute` are added.

Reply:
That's why. AsmDiff sucks at only showing the changed attribute. Instead, it shows everything as "-", then once again everything as "+", plus the actual difference where it applies.